sqlalchemy_risingwave/base.py

```diff
@@ -93,25 +75,37 @@ def get_columns(self, conn, table_name, schema=None, **kw):
             name, type_str = row[:2]
             # When there are type parameters, attach them to the
             # returned type object.
-            m = re.match(r"^(\w+(?: \w+)*)(?:\(([0-9, ]*)\))?$", type_str)
-            if m is None:
-                warn("Could not parse type name '%s'" % type_str)
-                typ = sqltypes.NULLTYPE
-            else:
-                type_name, type_args = m.groups()
+            m = re.match(r"^struct<([a-z ]+)>$", type_str)
+            if m:
+                sub_type = m.group(1)
```

Comment:
IIUC, the struct type can have multiple types inside it? But here seems we only process one

Reply:
Sorry, I forgot to add comma. Regular expression should be "struct<([a-z ,]+)>$"

Reply:
I mean, in a struct there may be mutiple types, but seems like we only process one type `sub_type`?

Reply:
My mistake here. I will fix it later.

Reply:
Yes. If it is non-trival to add `struct`, let's do it in other issues.

Reply:
I plan to change to this: If 'struct<' is at the beginning and '>' is at the ending, directly prompts that struct is not supported. Because nested structs are hard to parse.

Reply:
IIUC, this is same to reject all sturtc type, right?

Reply:
Yes. `struct` does not have a corresponding type in sql. Here I just want to check the legitimacy of struct type. But I found it is too complex.